### PR TITLE
add faucet output as var context

### PR DESCRIPTION
**Pull request by 0xpause**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- mpm faucet 输出交易信息和结果，可以用来获取相关信息，比如获取接受 faucet 的地址。`$.faucet[0].txn.raw_txn.decoded_payload.ScriptFunction.args[0]`

### CODE REVIEW

File: vm/starcoin-transactional-test-harness/src/lib.rs [hunk 1]
```diff
@@ -706,7 +706,7 @@ impl<'a> StarcoinTestAdapter<'a> {
         match output.output.status {
             TransactionStatusView::Executed => {
                 self.hack_account(addr)?;
-                Ok((None, None))
+                Ok((None, Some(serde_json::to_value(&output)?)))
```
Comment (jolestar) [inline vm/starcoin-transactional-test-harness/src/lib.rs:709]: output 里没有交易信息，包装成 `TransactionWithOutput`？

**Review by jolestar — APPROVED** [review]

**Comment by baichuan3** [comment]
从faucet中获取address给个示例呗

**Review by baichuan3 — APPROVED** [review]

File: vm/starcoin-transactional-test-harness/src/lib.rs (hunk 1 above)
Comment (0xpause) [reply vm/starcoin-transactional-test-harness/src/lib.rs:709]: 这个 output 就是 TransactionWithOutput

**Comment by 0xpause** [comment]
> 从faucet中获取address给个示例呗

前面 pr 说明里面有的

**Review by jolestar — APPROVED** [review]

**Pull request merged**